### DISCUSSION TITLE
What content would enrich user experience on hydrophone location pages

### DISCUSSION THREAD

**Discussion by scottveirs** [orcasound/orcasite#248] [Ideas]
Let's brainstorm and begin to prioritize what content could be added to the v3 drawer or the map to enrich the listening and learning experience of users as they explore one of the hydrophone location pages. For example, as of 10/26/23 this is the overall pagefor Sunset Bay:

<img width="1790" alt="Screenshot 2023-10-26 at 4 33 56 PM" src="https://github.com/orcasound/orcasite/assets/14044595/ffd8ca7e-3d9b-4728-922c-625ab72f72f4">

And this is the current drawer HTML content:

<img width="583" alt="Screenshot 2023-10-26 at 4 34 13 PM" src="https://github.com/orcasound/orcasite/assets/14044595/9e1991e5-e98f-451e-8091-faf5bb55aeeb">

Each drawer has a link to `Learn more` about the node that leads to the project page on the old Wordpress site.

**Comment by scottveirs** [comment]
One request (from Tom Dakin upon review of v3 recently) was depth and photos of the current deployment.

This could be met in part with extracts from the huge archive of visual assets we have in the shared Google photo albums we've maintained for each current site:

1. [Orcasound Lab](https://goo.gl/photos/gfwxYgtKp8EwkrhY9)
3. [North San Juan Channel](https://photos.app.goo.gl/fsgdBBd1UW4RyHhy7)
4. [Port Townsend](https://photos.app.goo.gl/RfJ4CTJhDYmavr2g8)
5. [Bush Point](https://photos.app.goo.gl/BEiynrzTPVxXKxkl2)
7. [Beach Camp at Sunset Bay](https://photos.app.goo.gl/SAvyHGEpvjr4jqGx5)
8. [MaST Center Aquarium](https://photos.app.goo.gl/NeGhWqDZtuzdWw426)
9. [Point Robinson](https://photos.app.goo.gl/PRgweMEMaHonWajTA)

As well as historic sites:

- [Neah Bay](https://photos.app.goo.gl/FZgz1SwocoTMAx8c8)
- [Seattle Aquarium ](https://photos.app.goo.gl/m7GBwX582H6Qj4tC9)
- [Lime Kiln](https://photos.app.goo.gl/FZgz1SwocoTMAx8c8) (2002-2012)

Most of these are linked to from deep within the project page for each location in the Wordpress site... but they could be surfaced much more prominently -- e.g. in a list of common assets presented as a footer in the drawer?

**Comment by Filadora** [comment]
Our team was planning to suggest a "recent activity" feature: an interactive list of sounds registered/shared by users recently.
